README.md

```diff
@@ -80,6 +80,8 @@ services:
       - ./:/home/node/app
     expose:
       - "8081"
+    ports: # use if it is necessary to expose the container to the host machine
+      - "8081": "8081"
```

Comment:
I thought it should be like this?
```suggestion
      - "8001:8001"
```

Reply:
I think that is indifferent, but I will take your recommendation.